api/api/models/taskmodels.py

```diff
@@ -1,16 +1,23 @@
 from django.db import models
+from api.models.elementmodels import Element
+from api.models.statusmodels import Status
 
 
 # Create your models here.
 class Task(models.Model):
     # Al momento de crearse una tarea, quiere decir que fue aprobada
     # por ende, la fecha de aprobacion es el dia en que se crea
     # y en consecuencia, se crea automaticamente
-    approved_date = models.DateTimeField(auto_now_add=True)
+    approved_date = models.DateField(auto_now_add=True)
 
     # La @realization_date tiene que ser asignada por el admin, es la
     # fecha en que se realiza el trabajo, por tanto no es autoasignada
+    # Formato: YYYY-MM-DD
+    realization_date = models.DateField()
+    fkAssignment_worker = models.IntegerField(default=0)
+    fkElement_id = models.IntegerField(default=0)
+    fkTask_status = models.IntegerField(default=0)
     realization_date = models.DateTimeField()
     fkAssignment_worker = models.IntegerField()

```

Comment:
Same, falta la relación de la llave foránea.

Reply:
Done

Reply:
LG2M